BUILDING.md

```diff
@@ -28,9 +28,8 @@ To build GMT, you have to install:
 - [curl](https://curl.haxx.se/)
 
 > Note for developers: It is recommended that you install the optional dependencies.
-> [Sphinx](http://www.sphinx-doc.org), [Sphinx-panels](https://sphinx-panels.readthedocs.io/en/latest/),
-> and [GraphicsMagick](http://www.graphicsmagick.org/) are required for building the documentation and running the
-> tests.
+> [GraphicsMagick](http://www.graphicsmagick.org/) is  required for building the documentation
+> and running the tests.
```

Comment:
Sphinx still requires separate installation, right? If so, I think it's worth keeping as a requirement in this note.

Reply:
Yes, fixed in 2f1e37ba775fbb59fb9485dc2c383d551838a31a